### PR TITLE
Fix names

### PR DESCRIPTION
**Pull request by iGom**
Hi, I fixed names to match patched roms from No-Intro Collection and N64 thumbnails from https://github.com/libretro-thumbnails/Nintendo_-_Nintendo_64/tree/55754e7b463c11d4b39ef6b1393f3897b3828905

### CODE REVIEW

File: metadat/no-intro/Nintendo - Nintendo 64.dat [hunk 1]
```diff
@@ -66,15 +66,15 @@ game (
 )
 
 game (
-	name "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07)"
-	description "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07)"
-	rom ( name "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07).z64" size 33554432 crc 7E718143 md5 0A5ACFEA0C7CF68AE25202040D5AD1EB sha1 3774E87AA383220060D330314F2C5BBB872F72CE )
+	name "40 Winks (Europe) (En,Es,It) (Proto)"
```
Comment (RobLoach) [inline metadat/no-intro/Nintendo - Nintendo 64.dat]: 40 Winks has the year at the end in No-Intro:
https://datomatic.no-intro.org/?page=show_record&s=24&n=0008

File: metadat/no-intro/Nintendo - Nintendo 64.dat [hunk 2]
```diff
@@ -210,15 +210,15 @@ game (
 )
 
 game (
-	name "AeroGauge (Japan) (Rev 1)"
-	description "AeroGauge (Japan) (Rev 1)"
-	rom ( name "AeroGauge (Japan) (Rev 1).z64" size 8388608 crc F322B641 md5 0620C2D134A0430F4AFA208FFEDA67B8 sha1 F7C2D4B0E77CC02DEAFE2E5FD95152B7E8BF86CF )
+	name "AeroGauge (Japan) (Rev A)"
```
Comment (RobLoach) [inline metadat/no-intro/Nintendo - Nintendo 64.dat]: AeroGauge uses Rev 1 in No-Intro
https://datomatic.no-intro.org/index.php?page=show_record&s=24&n=0025

**Comment by iGom** [comment]
Oh, I see now, so all changes to trow awyay. Yeah, at first I changed the names of the thumbnails to match database, but then I was convinced that the database is wrong..., sorry.

**Comment by RobLoach** [comment]
Nothing is lost. Learned where the broken thumbnails are! :smile: 

Thanks again for the help :+1: 

I'm re-building the packs right now, if you update a few N64 thumbnails, they'll get updated in the packs!

**Comment by RobLoach** [comment]
Oh, as an added sidenote. There's a thumbnails check list over here:

https://github.com/RobLoach/libretro-thumbnails-check/blob/master/out/Nintendo%20-%20Nintendo%2064.txt

**Pull request closed without merging**